### PR TITLE
8.0 calculator role

### PR DESCRIPTION
**Pull request by mathi123**
Superseeds #14

### CODE REVIEW

File: crm_construction_calculator/README.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Construction Calculator CRM
+===========
+
```
Comment (max3903) [inline crm_construction_calculator/README.rst:6]: '=' should underline the whole string to 'CRM'

**Comment by max3903** [comment]
:+1:

**Comment by JosDeGraeve** [comment]
:+1: 

just small detail when choosing xmlid's for in inherited views:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#inherited-xml

But that's to prove i did a thorough review more than anything else :)

**Comment by mathi123** [comment]
Super, thx! :smiley: :smiley:

**Pull request merged**